check.py

```diff
@@ -413,6 +413,7 @@ def fix(x):
     print s
     f = open('a.js', 'w')
     f.write(open(os.path.join(options.binaryen_bin, 'binaryen.js')).read())
+    f.write('var Binaryen = module.exports;\n')
```

Comment:
Hmm, good point, mozjs does not have CommonJS functionality. We could use node.js instead, but it would need a very recent node, we'd need to version-detect, etc. So all that seems out of scope for this PR I guess.